vitest.base.browser.config.ts

```diff
@@ -35,7 +48,7 @@ export default defineConfig({
       name: "chrome",
       headless: true,
       provider: "webdriverio",
-      slowHijackESM: false,
+      screenshotFailures: false,
```

Comment:
how are these two options related? and why don't we like screenshot failures? (whatever that means)

Reply:
They aren't related.

slowHijackESM seems to be a deprecated or removed option.

screenshotFailures will take a screenshot if there is a failure. Since we have no dom interaction, this doesn't make sense for us

Reply:
yeah, thought so, my comment mostly meant "can you please add some context here". It seems strange that the option is even a thing since we have headless set to true